packages/core/useRetry/index.md

```diff
@@ -0,0 +1,19 @@
+# useRetry
+
+Retry something until fulfilled.
+
+## Usage
+
+```ts
+import { useFetch, useRetry } from '@vueuse/core'
+
+const { onFinish } = useRetry(
+  () => useFetch('https://example.com/api/users'),
```

Comment:
I am not sure if that it's an ideal solution. If this function been called async, the composable will lost the context of current instance, which makes it failed to register cleanup functions and ends up with memory leak.

Reply:
So you have any hints or other VueUse examples were I can orient myself?